IPython/terminal/ipapp.py

```diff
@@ -102,6 +102,11 @@ def make_report(self,traceback):
         'Turn on auto editing of files with syntax errors.',
         'Turn off auto editing of files with syntax errors.'
 )
+addflag('simple-prompt', 'TerminalInteractiveShell.simple_prompt',
+        "Force simple minimal prompt using `raw_input`",
+        "Force simple usage of pompt_toolkit for interface.",
```

Comment:
s/pompt/prompt/

Also, it's not really 'simple'. Maybe 'Use a rich interactive prompt with prompt_toolkit'.